### PR TITLE
Adds primal farming implements.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Adds a rake (reskinned cultivator) and a wooden hatchet to the ashwalker base. Replaces their bucket with a wooden bucket.

Added crafting recipes for the rake and wooden bucket.

## Why It's Good For The Game

It looks more flavourful (inb4 where did they get the wood from) and adds a bit of functionality to unused content (i.e. ashwalker farming).

## Changelog
:cl:
add: Added wooden farming implements to the ashwalker base.
add: Makes rakes and wooden buckets craftable.
/:cl:

### CODE REVIEW

**Comment by Barhandar** [comment]
Make it mushroom-colored not wood-colored; one of the mushrooms on lavaland is already alleged to allow bowls made from it (i.e. is watertight and solid enough).

**Comment by Militaires** [comment]
ahem......

where did they get the wood from

**Comment by FantasticFwoosh** [comment]
If the tools aren't universally craftable from wood, this seems like a waste since there's a shed of metal boxes with _"stolen"_ supplies. You can also use mushroom bowls and barrels (tribal crafting) for liquid containers.

**Comment by nemvar** [comment]
My end goal is to remove that shed completely and replace it with ashwalker unique content. Currently it's just some random assortment of loot items for miners.

Also, here's the answer where they have the wood from. It's literally all over their base.
gotwood

**Comment by FantasticFwoosh** [comment]
Well ok then. I dont think they were being literal though.

Tribal crafting or wood/bone sheet crafting isn't racially locked to Ashwalkers so I look forward to seeing what comes out of your future changes.

**Comment by Militaires** [comment]
you'd think if ash walkers live in a barren lava planet they'd make shit using bones, rock/obsidian, mushroom and hide though, instead of actually growing a tree just so they can use its wood

**Comment by Daxxed** [comment]
Weeds aren’t really a problem and you can chop wood with a spear.
But anything that replaces the loot pinata with relevant equipment seems like a plus so I approve.

**Comment by habader** [comment]
this pr is kinda fucking pointless

you can chop logs up with a spear and remove weeds by building another dirt pipe already
most plants will grow fully before weeds become an issue unless youre growing gatfruit or soen

**Comment by nemvar** [comment]
You must be fun at parties.

**Comment by FantasticFwoosh** [comment]
>  and remove weeds by building another dirt pile already

Ash (reagent) from burning objects on the fire scooped up and applied removes passive weeds and feeds planted weeds (lavaland mushrooms have weed adaption traits).

**Comment by OverlyGeneric** [comment]
> My end goal is to remove that shed completely and replace it with ashwalker unique content. Currently, it's just some random assortment of loot items for miners.

Don't remove the combat upgrade it's for the ultimate goal of ashwalkers: subverting the AI and giving it a combat upgrade so it can kill everyone.

**Comment by ATH1909** [comment]
> > My end goal is to remove that shed completely and replace it with ashwalker unique content. Currently, it's just some random assortment of loot items for miners.
> 
> Don't remove the combat upgrade it's for the ultimate goal of ashwalkers: subverting the AI and giving it a combat upgrade so it can kill everyone.

This but unironically.

Also, it's a cool bit of loot that encourages miners to interact with ash lizards, which is nice.

**Comment by optimumtact** [comment]
You need to make them properly and give them crafting recipes

**Comment by Daxxed** [comment]
Ashliggers being able to subvert an AI is their main issue, they should be banned from using tech and be allowed back on station instead.
Lavaland AI can no longer interact with station so good luck.

**Comment by Tlaltecuhtli** [comment]
being allowed to go to the station will always end up on people spawning as lizard rushing to station and try to grief as much as they can with whatever knowledge they had from ghosting, you cant have free respawn to station and free antag at the same time

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 1]
```diff
@@ -93,6 +103,12 @@
 	playsound(src, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
 	return (BRUTELOSS)
 
+/obj/item/hatchet/wooden
```
Comment (FantasticFwoosh) [inline code/modules/hydroponics/hydroitemdefines.dm]: Wasn't anything supposed to be happening to this or did you just forget about it? No crafting recipie, no mention in changelog even with the effort to add sprites to [this pr's icons/obj/items_and_weapons.dmi](https://github.com/tgstation/tgstation/blob/6f60a8cbca2c54d7835bcec8862e9b70d91c101f/icons/obj/items_and_weapons.dmi) ?

Comment (nemvar) [reply code/modules/hydroponics/hydroitemdefines.dm]: I added it to the ashwalker base. That's about it.

That was the original vision of this PR, not to add more improvised weapons. I think it was mentioned in the original PR description, might have removed it by accident.

**Pull request merged**